sdk/cognitiveservices/cognitiveservices-luis-runtime/package.json

```diff
@@ -4,8 +4,10 @@
   "description": "LUISRuntimeClient Library with typescript type definitions for node.js and browser.",
   "version": "3.0.1",
   "dependencies": {
-    "@azure/ms-rest-js": "^1.8.1",
-    "tslib": "^1.9.3"
+    "@azure/ms-rest-js": "^2.0.3",
+    "@types/chai": "^4.2.0",
+    "@types/mocha": "^5.2.7",
```

Comment:
The 2 `@types` dependencies above are related to testing and so should be under `devDependencies`